LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Nice that this is finally added in. Should we capture the change somewhere on the PR comments, as it's not part of the ticket

Reply:
I have already added to the PR comment? Unless I've miss understood what you mean by this sorry.